.github/workflows/pr-smoke-testing.yaml

```diff
@@ -27,24 +28,18 @@ jobs:
           go install gotest.tools/gotestsum@latest
           go get -u github.com/proglottis/gpgme
 
-      - name: Install podman-compose
-        run: pip3 install podman-compose
+      - name: Create cluster
+        run: make cluster
+      
+      - name: Deploy
+        run: make deploy
 
-      - name: Install podman 4
+      - name: Check
         run: |
-          sudo mkdir -p /etc/apt/keyrings
-          curl -fsSL "https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key" \
-            | gpg --dearmor \
-            | sudo tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
-          echo \
-            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg]\
-              https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/ /" \
-            | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list > /dev/null
-          sudo apt-get update -qq
-          sudo apt-get -qq -y install podman
-
-      - name: Deploy
-        run: make build deploy
+          sleep 60
```

Comment:
We don't need this sleep anymore because of:
https://github.com/flightctl/flightctl/pull/91/commits/7aafc2f617951f6de1e38e7cda16ff83dc746f13#diff-dd530797092baca20b42189b556574346477aa6f9c99fd3ce82668d3ba4aee19R62